sources/packages/web/src/assets/css/base.scss

```diff
@@ -224,3 +224,7 @@ ul.no-bullets {
   color: $color-muted;
   font-weight: $font-weight-bold;
 }
+
+.search-box-display-width {
```

Comment:
Can you please elaborate on why this was needed?

Reply:
Without it, the UI looks like the screenshot below. This is cause by the "float-right" class in the action slot within the BodyHeader component. I set the width to be that same as what was set in figma and it fixed the issue.
![image](https://user-images.githubusercontent.com/72251620/174386900-c4e240c3-a0b6-4f3c-a5b8-7a92695645dd.png)